scripts/windows/sudo-install-wsl.ps1

```diff
@@ -0,0 +1,22 @@
+# //////////////////////////////////////////////////////////////////////
+# sudo-install-ws1.ps1
+
+param($Step)
```

Comment:
`ValidateSet` please. If we set [`Mandatory`](https://docs.microsoft.com/en-us/powershell/module/microsoft.powershell.core/about/about_functions_advanced_parameters?view=powershell-5.1#mandatory-argument) too, then we can skip the check below.

Reply:
Hmmm, I tried `Manadatory` but the parser didn't like it. Will retry.

Reply:
You're a good incentive for me to get something work after revisiting it.